EIPS/eip-663.md

```diff
@@ -13,7 +13,7 @@ requires: 3540, 5450
 
 ## Abstract
 
-Currently, `SWAP` and `DUP` instructions are limited to a stack depth of 16. Introduce two new instructions, `SWAPN` and `DUPN`, which lift this limitation and allow accessing the stack up to depth of 256 items.
+Currently, `SWAP*` and `DUP*` instructions are limited to a stack depth of 16. Introduce three new instructions, `SWAPN`, `DUPN` and `EXCHANGE` which lift this limitation and allow accessing the stack up to a depth of 32, 32 and 256 items, respectively.
```

Comment:
`32, 32 and 256` sounds off, shouldn't be `257, 256 and 33`? - however I'm wondering if this isn't too detailed. Maybe can consider making it vague "manipulating stack items at much greater depth"